.github/workflows/regression-tests.yml

```diff
@@ -168,6 +168,17 @@ jobs:
             cc: clang
             extra_name: "clang | "
 
+          - sim: icarus             # with gcov/gcovr
+            sim-version: v10_3
+            lang: verilog
+            python-version: 3.8
+            os: ubuntu-latest
+            extra_name: "gcovr | "
+            cxx: "ccache g++"
+            cc: "ccache gcc"
+            CFLAGS: -g --coverage
+            LDFLAGS: --coverage
```

Comment:
I think instead you want to feed these directly to tox below. Otherwise they will apply to the entire run (like building Icarus).

Reply:
Thanks, done.